doc/services/portability/posix/option_groups/index.rst

```diff
@@ -359,7 +359,7 @@ _POSIX_MESSAGE_PASSING
 
     mq_close(),yes
     mq_getattr(),yes
-    mq_notify(),
+    mq_notify(),yes
```

Comment:
Is it possible to get the doc update in a separate, 3rd commit?

Reply:
any preference? Should be first or last?

Reply:
@awojasinski - last ideally (3rd) - thanks!

Reply:
@cfriedt done